.github/workflows/full-integration-tests.yml

```diff
@@ -28,14 +28,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    # Sparse check out terra-helmfile repo version directories
-    - uses: ./.github/actions/checkout-helm-versions
-      with:
-        repository: broadinstitute/terra-helmfile
-        ref: master
-        token: ${{ secrets.BROADBOT_TOKEN }}
-        path: integration/terra-helmfile
-
     - name: Initialize Postgres DB for local server test run
```

Comment:
Successful run of `full-integration-tests` without this: https://github.com/DataBiosphere/terra-workspace-manager/actions/runs/7265787352/job/19796247576

Reply:
What was this action used for?

Reply:
I have no idea. It was added 2 years ago by Ivan (who is on vacation). I thought that deleting it would point out to me what it was needed for… but then everything passed.

Most likely the need for it went away with the various changes to how our services are configured/built/deployed. If deleting it causes some problem in the future, we can do more investigation.